sync-service/src/test/resources/application-test.yaml

```diff
@@ -0,0 +1,4 @@
+stayhome:
```

Comment:
I think this should use environment variables and not be defined in a static file.